examples/toy-chat/src/protocol.rs

```diff
@@ -37,6 +37,6 @@ impl Chat2Message {
     }
 
     pub fn timestamp(&self) -> DateTime<Utc> {
-        Utc.timestamp(self.timestamp as i64, 0)
+        Utc.timestamp_opt(self.timestamp as i64, 0).unwrap()
```

Comment:
Usually is better to use `expect` instead of `unwrap`, so you can add a meaningful message on why it should not panic:

```rust
Utc.timestamp_opt(self.timestamp as i64, 0).expect("Timestamps should not come out of bounds")
```

In this case we could probably just return a result as well, **why**? Because it looks like a "vulnerability" for the application, you only need to send an out of bounds timestamp to make the app panic (?)

Reply:
But well, it will break again with the `as i64` anyway 🤔 . For the sake of the example, lets just use `expect` 😸

Reply:
Awesome feedback, thanks 🙏. Updated.

Reply:
Let me try that locally...

Now I'm getting:

```
no method named `expect` found for enum `LocalResult` in the current scope
method not found in `LocalResult<DateTime<Utc>>`
```

Reply:
Oh, I see, I though `LocalResult` was just a `Result`, but it is actually better. 
Lets just return the `LocalResult<DateTime<Utc>>` then:

```rust
pub fn timestamp(&self) -> LocalResult<DateTime<Utc>> {
        Utc.timestamp_opt(self.timestamp as i64, 0)
    }
```

Reply:
Like

https://github.com/waku-org/waku-rust-bindings/pull/27/commits/cc2268fc4a6efce353d8b8377909a0d00c20aa75

and unwrap when formatting

https://github.com/waku-org/waku-rust-bindings/pull/27/commits/3d2eb4b04faecf835a18870d1e2b83d142d852bb

?

Sorry, I still got loads to learn 😅.

Reply:
yes, it should be enough with that 🚀